### ISSUE TITLE
Error: Can't add new command when connection is in closed stat

### ISSUE THREAD

**Issue by rubenhak** [kubevious/kubevious#25]
### Describe the bug
Error with reestablishing connection to mysql.

```
[2020-06-12 21:30:21.465 +0000] ESC[31mERRORESC[39m (MySqlDriver/1 on kubevious-6f59b8958f-4cz2p): ESC[36m[executeStatement] ERROR IN RULE_QESC[36m    at Connection._addCommandClosedState (/app/node_modules/mysql2/lib/connection.js:137:17)ESC[39m
ESC[36m    at PreparedStatementInfo.execute (/app/node_modules/mysql2/lib/commands/prepare.js:27:29)ESC[39m
ESC[36m    at /app/node_modules/kubevious-helpers/lib/mysql-driver.js:116:31ESC[39m
ESC[36m    at Promise._execute (/app/node_modules/bluebird/js/release/debuggability.js:384:9)ESC[39m
ESC[36m    at Promise._resolveFromExecutor (/app/node_modules/bluebird/js/release/promise.js:518:18)ESC[39m
ESC[36m    at new Promise (/app/node_modules/bluebird/js/release/promise.js:103:10)ESC[39m
ESC[36m    at /app/node_modules/kubevious-helpers/lib/mysql-driver.js:113:24ESC[39m
ESC[36m    at tryCatcher (/app/node_modules/bluebird/js/release/util.js:16:23)ESC[39m
ESC[36m    at Promise._settlePromiseFromHandler (/app/node_modules/bluebird/js/release/promise.js:547:31)ESC[39m
ESC[36m    at Promise._settlePromise (/app/node_modules/bluebird/js/release/promise.js:604:18)ESC[39m
ESC[36m    at Promise._settlePromiseCtx (/app/node_modules/bluebird/js/release/promise.js:641:10)ESC[39m
ESC[36m    at _drainQueueStep (/app/node_modules/bluebird/js/release/async.js:97:12)ESC[39m
ESC[36m    at _drainQueue (/app/node_modules/bluebird/js/release/async.js:86:9)ESC[39m
ESC[36m    at Async._drainQueues (/app/node_modules/bluebird/js/release/async.js:102:5)ESC[39m
ESC[36m    at Immediate.Async.drainQueues [as _onImmediate] (/app/node_modules/bluebird/js/release/async.js:15:14)ESC[39m
ESC[36m    at processImmediate (internal/timers.js:456:21)ESC[39m
Unhandled rejection Error: Can't add new command when connection is in closed state
    at Connection._addCommandClosedState (/app/node_modules/mysql2/lib/connection.js:137:17)
    at PreparedStatementInfo.execute (/app/node_modules/mysql2/lib/commands/prepare.js:27:29)
    at /app/node_modules/kubevious-helpers/lib/mysql-driver.js:116:31
    at Promise._execute (/app/node_modules/bluebird/js/release/debuggability.js:384:9)
    at Promise._resolveFromExecutor (/app/node_modules/bluebird/js/release/promise.js:518:18)
    at new Promise (/app/node_modules/bluebird/js/release/promise.js:103:10)
    at /app/node_modules/kubevious-helpers/lib/mysql-driver.js:113:24
    at tryCatcher (/app/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/app/node_modules/bluebird/js/release/promise.js:547:31)
    at Promise._settlePromise (/app/node_modules/bluebird/js/release/promise.js:604:18)
    at Promise._settlePromiseCtx (/app/node_modules/bluebird/js/release/promise.js:641:10)
    at _drainQueueStep (/app/node_modules/bluebird/js/release/async.js:97:12)
    at _drainQueue (/app/node_modules/bluebird/js/release/async.js:86:9)
    at Async._drainQueues (/app/node_modules/bluebird/js/release/async.js:102:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/app/node_modules/bluebird/js/release/async.js:15:14)
    at processImmediate (internal/timers.js:456:21)
```

**Comment by kubevious** [comment]
Same as https://github.com/kubevious/kubevious/issues/12

**Comment by kubevious** [comment]
duplicate

**Comment by kubevious** [comment]
Resolved in [v0.7](https://github.com/kubevious/kubevious/releases/tag/v0.7).
